components/ingest-service/grpc/grpc.go

```diff
@@ -45,7 +48,7 @@ func Spawn(opts *serveropts.Opts) error {
 
 	var (
 		grpcServer  = opts.ConnFactory.NewServer()
-		A1migration = migration.New(client)
+		migrationer = migration.New(client)
```

Comment:
Updated the variable name because it is not only for A1 migrations.

Reply:
This might fail the eventual spellcheck run. `migrator` is used in a few places though.

Reply:
I thought `migrator` was a word https://en.oxforddictionaries.com/definition/migrator

Reply:
That's what I meant, I think `migrator` is a word but I'm not sure `migrationer` is.

Reply:
Thanks, I did not see that. I am updating now.